apps/meteor/client/views/room/components/body/composer/messageBox/MessagePreview.tsx

```diff
@@ -0,0 +1,64 @@
+import { css } from '@rocket.chat/css-in-js';
+import { Box, MessageBody, Palette } from '@rocket.chat/fuselage';
+import { Markup } from '@rocket.chat/gazzodown';
+import React from 'react';
+import type { ReactElement } from 'react';
+
+import GazzodownText from '../../../../../../components/GazzodownText';
+
+type MessagePreviewProps = {
+	md: any;
+	channels: any[];
+	mentions: any[];
+};
+
+export const MessagePreview = ({ md, channels, mentions }: MessagePreviewProps): ReactElement => {
```

Comment:
Message Preview Component. Code is similar to (apps/meteor/client/components/message/MessageContentBody.tsx). Used already existing code so that preview looks exactly similar to the sent message display. 

Todo:
- [ ] Fix types.